FHIR/vscode-fsh
# Update FSH parser+lexer and antlr4 dependency

**Description:**
New versions of the antlr4 package and type definitions are used. Update the FSH parser and lexer to the newest SUSHI grammar that was generated with that version of antlr4.

Rules for alias and paramRuleSet are changed. Use new child rules to get entity details.

Resolve with null instead of rejecting when providing definition. The result is the same, but it avoids filling up the extension host logs.

**Testing Instructions:**
No test definitions are changed as a result of this update. Confirm that all tests pass on your system. Make use of the extension's features (particularly provide definition, as that relies on the parser) in a FSH project.

**Related Issue:**
Issue #79 is already closed, but the antlr4 dependency was not updated along with the other dependencies.
